app/src/pages/inside/historyPage/historyToolbar/actionPanelWithGroupOperations/actionPanelWithGroupOperations.jsx

```diff
@@ -293,14 +287,16 @@ export class ActionPanelWithGroupOperations extends Component {
   handleEditDefects = (eventData) => {
     const { selectedItems, debugMode, onEditDefects, tracking } = this.props;
     const items = eventData && eventData.id ? [eventData] : selectedItems;
+
     tracking.trackEvent(
       HISTORY_PAGE_EVENTS.MAKE_DECISION_MODAL_EVENTS.getOpenModalEvent(
         items.length === 1
-          ? items[0].issue.issueType.startsWith(TO_INVESTIGATE_LOCATOR_PREFIX)
+          ? items[0].issue && items[0].issue.issueType.startsWith(TO_INVESTIGATE_LOCATOR_PREFIX)
           : undefined,
         'actions',
       ),
     );
+
     onEditDefects(items, {
       fetchFunc: this.unselectAndRefreshItems,
       debugMode,
```

Comment:
Looks like in this case `unlinkIssueEvents`, `postIssueEvents`, `linkIssueEvents` need to be parametrized with `MAKE_DECISION` parameter.
On this page there are two options to open `link/post/unlink` modals:
- Directly via list of actions;
- Via Make decision modal.

Refer to how you did it for `StepPage`.